wire-ios-data-model/Source/E2EIdentity/GracePeriodRepository.swift

```diff
@@ -18,15 +18,15 @@
 
 import Foundation
 
+/// The repository is responsible for storing the grace period during which the user must enroll the end-to-end identity certificate.
+/// The grace period starts at the moment where the client fetches and processes the team feature flag.
 public protocol GracePeriodRepositoryInterface {
 
     func fetchEndGracePeriodDate() -> Date?
```

Comment:
optional: I would call it `gracePeriodEndDate`, not `endGracePeriodDate`.

Reply:
Sorry, I forgot. I pushed the changes